code/to_bsdd.py

```diff
@@ -57,18 +56,45 @@ def skip_by_package(element):
     
 HTML_TAG_PATTERN = re.compile('<.*?>')
 MULTIPLE_SPACE_PATTERN = re.compile(r'\s+')
-CHANGE_LOG_PATTERN = re.compile(r'\{\s*\.change\-\w+\s*\}.+', flags=re.DOTALL)
+# CHANGE_LOG_PATTERN = re.compile(r'\{\s*\.change\-\w+\s*\}.+', flags=re.DOTALL)
+CURLY_BRACKET_PATTERN = re.compile('{.*?}')
+
 def strip_html(s):
+    try:
+        s1 = html.unescape(s)
+    except TypeError:
+        # error when name contains link, for example: "https://github.com/buildingSMART/IFC4.3.x-development/edit/master/docs/schemas/core/IfcProductExtension/Types/IfcAlignmentTypeEnum.md#L0 has no content"
+        s1 = ''
+    i = s1.find('\\n')
+    if i != -1:
+        s1 = s1[:i]
+    s2 = re.sub(HTML_TAG_PATTERN, ' ', s1)
+    s3 = re.sub(CURLY_BRACKET_PATTERN, ' ', s2)
+    s4 = re.sub('   ', ' ', s3)
+    s5 = re.sub('  ', ' ', s4)
+    return s5
+
+def split_at_word(w, s):
+    if w in s.lower():
+        if not w+"ing" in s.lower():
```

Comment:
This is called reverse stemming. Did you consider sth like https://pypi.org/project/reversestem/ Haven't tested, doesn't look very maintained. But beats `+"ing"` I guess.

Reply:
good point. I found PorterStemmer from nltk.

Reply:
added reverse stemming